crawl-ref/source/output.cc

```diff
@@ -1715,7 +1715,8 @@ static void _print_next_monster_desc(const vector<monster_info>& mons,
         {
             int desc_colour;
             string desc;
-            mons[start].to_string(count, desc, desc_colour, zombified);
+            mons_to_string_pane(desc,desc_colour,zombified,mons,start,count);
```

Comment:
White space here, space out the commas; and delete the old line, instead of leaving it in a comment.